templates/download/cloud/index.html

```diff
@@ -95,9 +98,10 @@ <h3 class="list-stepped__title">Install MAAS</h3>
                     </p>
                     <ul class="no-bullets">
```

Comment:
Section 3 ("Install MAAS") doesn't have bullet points but the other steps do. Should the styling be aligned?